src/services/pwabuilder.js

```diff
@@ -24,12 +25,16 @@ PWABuilder.prototype.createManifestFromUrl = function (url, client) {
   }
 
   return Q.Promise(function (resolve, reject) {
-    var callback = function (err, manifestInfo) {
+    // Attempt to use the azure function
```

Comment:
Definitely some tech debt to cleanup this code, especially with regards to Q and nested promises. Async and await will be our friends. 😊